Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -2453,21 +2456,21 @@ protected virtual void UpdateAxleDriveForce()
         /// <summary>
         /// This function applies a sign to the motive force as a function of the direction of the train.
         /// </summary>
-        protected virtual void ApplyDirectionToTractiveForce()
+        protected virtual void ApplyDirectionToTractiveForce(ref float tractiveForceN, int numberofengine )
```

Comment:
Why not make the last parameter optional? The calls to it would be simpler.
```suggestion
        protected virtual void ApplyDirectionToTractiveForce(ref float tractiveForceN, int engineIndex = 0 )
```

Reply:
Revised as suggested